### DISCUSSION TITLE
请问如何使用微信小程序的xr-frame?

### DISCUSSION THREAD

**Discussion by Sweechirly** [NervJS/taro#14335] [Q & A] [answered]
目前编译以后模板会报错

**Comment by Sweechirly** [comment]
没得人啊

**Reply by HaroGo** [reply, marked as answer]
`@taro/plugin-inject` 可能不太好使，可以参考 [taro-sample-weapp](https://github.com/NervJS/taro-sample-weapp/tree/next) 中对原生模块的引用

**Reply by Sweechirly** [reply]
好的，感谢

**Comment by KitStarLee** [comment]
兄弟，你接入成功了么？看了半天，还是不会

**Comment by KitStarLee** [comment]
@Sweechirly @HaroGo  我是 新手，taro的案例也看不懂，毕竟案例接入的也不是 xr-frame。加上没有文档说明。完全看不懂

**Reply by KitStarLee** [reply]
@smoothdvd @guotie 你好，有什么方案么，如果不能添加 xr-frame 的话，我就不深入看了。

**Reply by Sweechirly** [reply]
> @smoothdvd @guotie 你好，有什么方案么，如果不能添加 xr-frame 的话，我就不深入看了。

在taro里使用原生微信小程序的模板就可以了

**Reply by KitStarLee** [reply]
感谢回复，是直接用 xr-frame 模版下面的所有 components文件夹内容么？里面的 文件需要进行处理么？还是可以直接使用？ 我目前用的 ts + react。使用的时候，总是不行。

**Reply by Sweechirly** [reply]
> 感谢回复，是直接用 xr-frame 模版下面的所有 components文件夹内容么？里面的 文件需要进行处理么？还是可以直接使用？ 我目前用的 ts + react。使用的时候，总是不行。

https://docs.taro.zone/docs/hybrid 先放一个试试

**Reply by KitStarLee** [reply]
谢了，我测了下。是用 ts 就不行。JS 就可以。同样的代码，同样是 发布后都没 components/common 文件夹。但 ts报错无法执行，js 可以执行，并使用。

**Reply by KitStarLee** [reply]
你用的是 ts 还是 js？

**Comment by KitStarLee** [comment]
@Sweechirly 找到问题了，留言一下，帮助后面的人。 是 使用了 Vite 编译 就报一些错误。然后改了 Webpack 就好了。目前不太知道 如何配置 Vite 可以让正确，所以先不用Vite 了。但是 Vite 速度真的快，后面有人找到解决办法了，请告知，感谢
